templates/vue/src/components/lightbox/WpPostMedia.vue

```diff
@@ -26,6 +29,8 @@ export default {
       loading: true,
       title: "",
       content: "",
+      scrollHeight: 0,
+      clientHeight: 0,
```

Comment:
This has to be added to the `data` field of the parent component.

Reply:
Can you expand? Is there an action you'd like me to take on this?

Reply:
No - please see comment below.